### PR TITLE
SiteFilterBehavior fixed

### PR DESCRIPTION
**Pull request by pedrofs**
There was a trying to access a non existent index due to a empty
sites table when it tries to get the currentSite. I've fixed it with a
simple check on the currentSite method returning.

### CODE REVIEW

**Comment by pedrofs** [comment]
This commit fixes the issue https://github.com/rchavik/sites/issues/8

File: Model/Behavior/SiteFilterBehavior.php [hunk 1]
```diff
@@ -36,7 +36,12 @@ public function beforeFind(Model $model, $query) {
 		}
 		$this->_setupRelationships($model, $this->settings[$model->alias]);
 		$site = Sites::currentSite();
-		$sites = array_unique(array(Sites::ALL_SITES, $site['Site']['id']));
+
+		$sites = array(Sites::ALL_SITES);
+
+		if ($site) {
+			array_unique(array(Sites::ALL_SITES, $site['Site']['id']));
```
Comment (rchavik) [inline Model/Behavior/SiteFilterBehavior.php]: shouldn't this be: 

``` php
$sites = array_unique(array(Sites::ALL_SITES, $site['Site']['id']));
```

**Comment by rchavik** [comment]
Also, please adjust your commit message.  "SiteFilterBehavior fixed" is a little vague as it does not explained what was fixed.  Maybe: "Fix: SiteFilterBehavior raises error when no site is configured"

Check out Croogo's: https://github.com/croogo/croogo/blob/master/CONTRIBUTING.mdown#making-changes

**Comment by pedrofs** [comment]
Ok. I will improve the message as you've said.

You are sure. Big mistake from my side. I've forgotten the assignment on that line hehe. I'll fix that.

Thanks.

**Comment by pedrofs** [comment]
@rchavik it is done!

**Pull request closed without merging**